test_scripts/API/ServiceStatusUpdateToHMI/026_Video_service_protected_start_stream_NONE_hmi_level.lua

```diff
@@ -0,0 +1,62 @@
+-----------------------------------------------------------------------------------------------------------------------
+-- Proposal:
+-- https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0211-ServiceStatusUpdateToHMI.md
+-----------------------------------------------------------------------------------------------------------------------
+-- Description: Attempt to open protected Video/Audio service with OnServiceUpdate notification in case
+-- App start stream in NONE HMI level
+--
+-- Preconditions:
+-- 1) SDL certificate is missing/expired
+-- 2) App is registered with NAVIGATION appHMIType
+-- Steps:
+-- 1) App sends StartService request (<service_type>, encryption = true)
+-- SDL does:
+--   - send OnServiceUpdate (<service_type>, REQUEST_RECEIVED) to HMI
+--   - send OnServiceUpdate (<service_type>, REQUEST_REJECTED) to HMI
+--   - send StartServiceNACK(<service_type>, encryption = false) to App
+-----------------------------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/ServiceStatusUpdateToHMI/common')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Constants ]]
+local serviceId = 11
+
+--[[ Local Functions ]]
+function common.onServiceUpdateFunc(pServiceTypeValue)
+  common.getHMIConnection():ExpectNotification("BasicCommunication.OnServiceUpdate",
+    { serviceEvent = "REQUEST_RECEIVED", serviceType = pServiceTypeValue, appID = common.getHMIAppId() },
+    { serviceEvent = "REQUEST_REJECTED", serviceType = pServiceTypeValue, appID = common.getHMIAppId() })
+  :Times(2)
+end
+
+function common.serviceResponseFunc(pServiceId)
+  common.getMobileSession():ExpectControlMessage(pServiceId, {
+    frameInfo = common.const.FRAME_INFO.START_SERVICE_NACK,
+    encryption = false
+  })
+end
+
+function common.policyTableUpdateFunc()
+  common.getHMIConnection():ExpectNotification("SDL.OnStatusUpdate")
+  :Times(0)
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Init SDL certificates", common.initSDLCertificates,
+  { "./files/Security/client_credential_expired.pem", true })
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("App registration", common.registerAppWOPTU)
```

Comment:
@TMyza In case if PTU is going to be done just after app registration, app has to be registered only with `common.registerApp()` function

Reply:
Corrected in 59e7a46